src/plugins/data-card/data-card-tile.tsx

```diff
@@ -40,6 +47,84 @@ export const DataCardToolComponent: React.FC<ITileProps> = observer((props) => {
     }
   }, [content, model.id, onRequestUniqueTitle]);
 
+  /* ==[ Drag and Drop ] == */
+
+  const handleDragOver = (e: React.DragEvent<HTMLDivElement>) => {
+    const isAcceptableDrag = isAcceptableDataCardDrag(e);
+    onSetCanAcceptDrop(isAcceptableDrag ? model.id : undefined); //this turns off highlighting outer edge
+  };
+
+  const isAcceptableDataCardDrag =  (e: React.DragEvent<HTMLDivElement>) => {
+    const draggingWithinItself = ui?.selectedTileIds.includes(model.id);
+    if (draggingWithinItself){
+      setHighlightDataCard(false);
+      return false;
+    }
+
+    const numTilesDragged = ui?.selectedTileIds.length;
```

Comment:
If the tile is being dragged from another window, there might be 0 tiles selected in the current CLUE window. Rather than looking at the selectedTileIds, you should look at the contents of the dataTransfer object to see how many tiles are being dragged.

Based on `handleTileDragStart`, there will only be a tile type entry in the dataTransfer if there is a single tile. So I'd expect `extractDragTileType` to return undefined if there is more than one tile being dragged. So I'd recommend checking for that instead of looking at the length o selectedTileIds.